runtime/src/accounts_db.rs

```diff
@@ -9558,6 +9558,11 @@ pub mod tests {
                 LoadZeroLamports::SomeWithZeroLamportAccountForTests,
             )
         }
+
```

Comment:
same fn added in a separate pr. The code calling this fn is a different pattern in this pr.